client/app/bundles/users/components/Widget.tsx

```diff
@@ -0,0 +1,47 @@
+import { ReactNode } from 'react';
+import { Typography } from '@mui/material';
+
+interface ContainerProps {
+  children?: ReactNode;
+  className?: string;
+}
+
+interface WidgetProps extends ContainerProps {
+  title: string;
+  subtitle?: string;
+}
+
+const Widget = (props: WidgetProps): JSX.Element => (
+  <form
```

Comment:
Because there's no need to. It's just a couple of fields and `useState` is enough for the job. Using `Form` brings so much burden of react-hook-form, re-renders for every value change, etc.